app/views/downloads/guis/index.html.haml

```diff
@@ -124,6 +124,11 @@
           %p.description
             <strong>Platforms:</strong> Windows, Mac, Linux</br>
             <strong>Price:</strong> $29/user (Free 15 day trial)
+          =link_to(image_tag('guis/tortoisegit@2x.png', {:width => '294', :height => '166'}), "https://tortoisegit.org/")
```

Comment:
Hmm, I think this is missing the necessary `li` tag to mark its platforms. Looks like the entry above is missing it, too, which is probably what you used as a template.